dags/local_automated_ampliseq.py

```diff
@@ -0,0 +1,153 @@
+from airflow.models import DAG
+from airflow.operators.bash import BashOperator
+from airflow.operators.python import PythonOperator
+from airflow.decorators import task
+from airflow.utils.task_group import TaskGroup
+import pendulum
+import csv
+import os
+import textwrap
+from datetime import timedelta
+
+# Constants for file paths
+BASE_PATH = "<PATH TO LOCAL DATA FOLDER>"
+PROVENANCE_PATH = os.path.join(BASE_PATH, "processed_studies_provenance.csv")
+ALL_STUDIES_PATH = os.path.join(BASE_PATH, "amplicon_studies.csv")
+CONFIG_PATH = os.path.join(BASE_PATH, "ampliseq.config")
+AMPLISEQ_VERSION = '2.9.0'
+
+def create_dag():
```

Comment:
so if were going to have two ampliseq dags (which id rather avoid but wont push you on, so long as things work and you and dan are both happy).. but if we are going to have two. they should probably be refactored some. we want to avoid duplicated code as much as possible in a production system, bc its impossible to maintain (ask me how i know lol). so id recommend considering some alternatives:
1. writing some of these tasks that you mean to reuse across both dags into a separate file, wrapping them in a method like 'make_task_X' and have it return a reference to a task
2. looking into TriggerDagRunOperator
3. just writing a single dag using a loop to generate the tasks necessary for each study